composer.json

```diff
@@ -80,17 +80,17 @@
 			"@ci:composer:require-checker"
 		],
 		"ci:composer:normalize": "@composer normalize --dry-run",
-		"ci:composer:require-checker": "composer-require-checker",
+		"ci:composer:require-checker": "@php composer-require-checker",
 		"ci:composer:validate": "@composer validate",
 		"ci:php": [
 			"@ci:php:lint",
 			"@ci:php:cs",
 			"@ci:php:stan",
 			"@ci:php:unit"
 		],
-		"ci:php:cs": "php-cs-fixer fix -v --dry-run --using-cache no --diff",
-		"ci:php:lint": "phplint --no-progress",
-		"ci:php:stan": "phpstan analyse --no-progress",
+		"ci:php:cs": "@php php-cs-fixer fix -v --dry-run --using-cache no --diff",
+		"ci:php:lint": "@php phplint --no-progress",
+		"ci:php:stan": "@php phpstan analyse --no-progress",
 		"ci:php:unit": "phpunit",
```

Comment:
A bug in Composer 1 currently avoids to use `@php` here too. Must be done once Composer 1 isn't removed from CI.